packages/teleport/src/services/auth/makeRecoveryCodes.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2021 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { RecoveryCodes } from './types';
+
+export function makeRecoveryCodes(json): RecoveryCodes {
```

Comment:
maybe add a comment here that explains that this only expects a response when it's with a cloud user? might be confusing to see this in OSS